components/BusinessLogo.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="elevation-3 business-logo">
+    <img
+      :alt="logoInfo.logoFallback"
+      :src="logoInfo.logoPath || require('@/static/images/fallback_logo.jpg')"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'BusinessLogo',
+
+  props: {
+    logoInfo: {
```

Comment:
as mentioned in business list, i would pass the whole business object and do all the data extraction (alt, img src) inside this component